packages/nodejs/src/logger.ts

```diff
@@ -0,0 +1,71 @@
+import winston from "winston"
+const { combine, timestamp, printf } = winston.format
+
+export class Logger {
+  type: string
+  level: string
+  logger: winston.Logger
+
+  constructor(type: string, level: string, filename?: string) {
+    this.type = type
+    this.level = this.translateLogLevel(level)
+
+    let transport
+
+    if (type == "file") {
+      transport = new winston.transports.File({ filename: filename })
+    } else {
+      transport = new winston.transports.Console()
+    }
+
+    const logFormat = printf(({ level, message, timestamp }) => {
+      return `[${timestamp} (process) #${process.pid}][appsignal][${level}] ${message}`
```

Comment:
Yep, I decided to keep it here to avoid adding more logic to the Logger. Do we really need to add that bit only to the stdout logger 🤔 ? I can't see a benefit for it.

Reply:
I don't really mind either way. I see how consistency with the format used by the agent and extension is a nice to have, but I don't think this is a deal breaker.

Reply:
The `[appsignal]` prefix was added back in the day to more easily be able to tell from which system the message comes from, in what can be a mix of different log messages.